assets/js/modules/analytics-4/datastore/webdatastreams.js

```diff
@@ -462,15 +462,17 @@ const baseSelectors = {
 			const info = {};
 			const propertyIDs = [];
 
-			summaries.forEach( ( { _id: accountID, propertySummaries } ) => {
-				propertySummaries.forEach( ( { _id: propertyID } ) => {
-					propertyIDs.push( propertyID );
-					info[ propertyID ] = {
-						accountID,
-						propertyID,
-					};
-				} );
-			} );
+			summaries.forEach(
+				( { _id: accountID, propertySummaries = [] } ) => {
```

Comment:
This fallback to the empty array was the main initial fix as the key isn't always defined in the `accountSummary` if there are no properties. This is no longer the case however as `populateAccountSummaries` will ensure every AS has `propertySummaries` defined.